tajo-common/src/main/java/org/apache/tajo/conf/TajoConf.java

```diff
@@ -203,6 +203,7 @@ public static int setDateOrder(int dateOrder) {
 
     // Query Configuration
     QUERY_SESSION_TIMEOUT("tajo.query.session.timeout-sec", 60, Validators.min("0")),
+    QUERY_SESSION_CACHE_SIZE("tajo.query.session.cache-size", 1000000, Validators.min("1000000")),
```

Comment:
Hi, @jihoonson . Thank you for kind advice.
* What about *tajo.query.session.query-cache-size-mb*, then? I think it would be more proper according to your advice.
* The minimum cache size is *1MB* in terms of SQL String length. As you see in TAJO-1430 example, 100K-length query takes over 30 seconds in SQL parsing, so the cache will hold up to 10 x 100K queries accessed in last 1 hour. In real situation, 10 or more MB is needed.
* For the cache on/off feautre, is it okay by using the condition *tajo.query.session.query-cache-size-mb=0*?